acme/LICENSE.txt

```diff
@@ -0,0 +1,190 @@
+   Copyright 2015 Internet Security Research Group
```

Comment:
Can we get away with symlinking all the copies of the LICENSE file in the tree, rather than proliferating them in git?  Or copying them into each subtree during release?